src/components/spatial/Spatial.js

```diff
@@ -66,9 +65,13 @@ export default function Spatial(props) {
     molecules = {},
     cells = {},
     neighborhoods = {},
+    neighborhoodsOn = false,
```

Comment:
Looks good. Only minor comment would be to update this `neighborhoodsOn` to match the `are___On` convention

Reply:
Good catch.